types/angular/index.d.ts

```diff
@@ -2183,6 +2183,8 @@ declare namespace angular {
             has(name: string): boolean;
             instantiate<T>(typeConstructor: {new(...args: any[]): T}, locals?: any): T;
             invoke<T = any>(func: Injectable<Function | ((...args: any[]) => T)>, context?: any, locals?: any): T;
+            loadNewModules(modules: Array<IModule|string|Injectable<(...args: any[]) => string>>): void;
```

Comment:
1) it should be `Injectable<(...args: any[]) => void>`, not `Injectable<(...args: any[]) => string>`

2) please add jsdoc

Reply:
PTAL. (Please take a look).